test/e2e/tests/eg_upgrade.go

```diff
@@ -24,6 +24,10 @@ import (
 	"sigs.k8s.io/gateway-api/conformance/utils/suite"
 )
 
+func init() {
+	UpgradeTests = append(UpgradeTests, EGUpgradeTest)
```

Comment:
something similar is also needed for shutdown ?

Reply:
this has already been included through #2839